config/file/parser.go

```diff
@@ -192,10 +192,7 @@ type unmarshaledConfig struct {
 		QueryParams map[string]string   `yaml:"queryParams" json:"queryParams"`
 		Header      map[string][]string `yaml:"header" json:"header"`
 		Timeout     *string             `yaml:"timeout" json:"timeout"`
-		Body        struct {
-			Type    *string                 `yaml:"type" json:"type"`
-			Content *map[string]interface{} `yaml:"content" json:"content"`
-		} `yaml:"body" json:"body"`
+		Body        string              `yaml:"body" json:"body"`
```

Comment:
I think @GregoryAlbouy suggested something like 

```yml
body:
  - type: "raw"
  - content: '{"key":"value"}'
```

This is way easier however. I think it's good enough. Everyone ok ?

Reply:
Yep, the condensed syntax was intended for CLI usage only because of its specific constraints, in config files it makes more sense to use the explicit one.